spec/support/factory_girl.rb

```diff
@@ -16,6 +29,7 @@
     f.zip_code "02125"
     f.website_url { Faker::Internet.url }
     f.phone { Faker::Number.number(10) }
+    f.user_id "1"
```

Comment:
So, two things here - you don't need the `|f|` or the `f.` anywhere in this whole thing, you can just define it without that variable like you did for the `user` factory. 

Also, read up on creating factories with associated objects - I have gist notes [here](https://gist.github.com/cmkoller/1dcc8815669d02b5a793). You shouldn't be passing in a value for user_id here - instead, you should be asking FactoryGirl to create a user object in the db and associate it for you. You can do that by just putting `user` on line 32, and nothing else. (Do read the notes I provided though, it's important to understand!)